### PR TITLE
add dummy blockdevice, and inherited Tmpfs and Proc

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: src/main/perl/Proc.pm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#${PMpre} NCM::Proc${PMpost}
+
+use parent qw(NCM::DummyBlockdevice);
+
+sub devpath
+{
+    return "tmpfs";
```
Comment (ned21) [inline src/main/perl/Proc.pm:7]: Shouldn't this be "proc" not tmpfs?

**Review by ned21 — COMMENTED** [review]
I commented on https://github.com/quattor/template-library-core/pull/207/ that I wasn't sure if dummy was quite correct, but seeing it in this context then I'm probably OK with it. We do end up needing an NCM::Blockdevice object, it's not as empty as I first thought.

File: src/main/perl/Proc.pm (hunk 1 above)
Comment (jrha) [reply src/main/perl/Proc.pm:7]: @stdweird?

Comment (jrha) [reply src/main/perl/Proc.pm:7]: EL6 - `proc on /proc type proc (rw)`
EL7 - `proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)`
EL8 - `proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)`

**Comment by stdweird** [comment]
@jrha @ned21 last commit has some graceful workaround

**Comment by stdweird** [comment]
@jrha tests pass now

**Comment by jrha** [comment]
@stdweird thanks, that odd behaviour might also explain some other odd failures we've seen in the GitHub actions.

**Review by jrha — APPROVED** [review]

**Pull request merged**